hack/observability/visualizer/kustomization.yaml

```
@@ -7,7 +7,7 @@ helmCharts:
     releaseName: visualizer
     namespace: observability
     valuesFile: values.yaml
-    version: 1.4.0
+    version: 1.4.1
```

```suggestion
    version: 1.4.0
```

This still needs 1.4.0, the release is 1.4.1 but the helm chart is 1.4.0

---

Sure - let me make the change